.github/workflows/build_ruby.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Build and package Ruby runtime
         run: bin/build_ruby "$STACK" "${{inputs.ruby_version}}"
       - name: Verify ruby executable and output rubygems version
-        run: bin/print_gem_version "$STACK" "${{inputs.ruby_version}}"
+        run: bin/print_summary "$STACK" "${{inputs.ruby_version}}" >> $GITHUB_STEP_SUMMARY
```

Comment:
It's not showing for some reason :/

https://github.com/heroku/docker-heroku-ruby-builder/actions/runs/8527371617/job/23358666172

<img width="2672" alt="image" src="https://github.com/heroku/docker-heroku-ruby-builder/assets/59744/33ce2182-adc1-4451-89e6-682dfa992c93">

Reply:
It also hides the contents. We should probably `tee` or make sure it shows in the output even if the summary feature doesn't work.

Reply:
It is available on the summary page, not the individual action page